core/runtime/src/main/java/org/eclipse/sw360/antenna/util/LicenseSupport.java

```diff
@@ -98,4 +98,14 @@ public static LicenseInformation fromSPDXExpression(SpdxExpression spdxExpressio
         }
         throw new ExecutionException("SPDX expression=[" + spdxExpression.toString() + "] could not be parsed");
     }
+
```

Comment:
The following code will collide with Onur, please talk

Reply:
It's just a new method and I think my changes will be merged after this PR, so i can adapt it during the rebase.